app/e2e/common/pages/playwright-landing-page.ts

```diff
@@ -0,0 +1,158 @@
+// playwright-landing-page.ts
```

Comment:
Why is this named playwright landing page? If it's for the landing page, then it can just be named `landing.ts`. (page is repetitive since you're already under that dir.)

Reply:
I was just following the naming convention that was in the playwright [docs](https://playwright.dev/docs/test-pom). I didn't want to make it the same name as the page so I could avoid confusion when searching for a particular file but we can discuss different naming conventions.

Reply:
resolved in meeting.